typer/main.py

```diff
@@ -181,7 +181,7 @@ def callback(
         help: Optional[str] = Default(None),
         epilog: Optional[str] = Default(None),
         short_help: Optional[str] = Default(None),
-        options_metavar: str = Default("[OPTIONS]"),
+        options_metavar: Optional[str] = None,
```

Comment:
Should this not use the `Default` class?
```suggestion
        options_metavar: Optional[str] = Default(None),
```